tests/framework/CodeInterfaceTests/test_generic_interface.xml

```diff
@@ -27,6 +27,7 @@
     <Input name="one.xml" type="">one.xml</Input>
     <Input name="inp.two" type="">inp.two</Input>
     <Input name="mesh" type="">dummy.e</Input>
+    <Input name="a_dummy_file_for_subdirectory" type="" subDirectory="testSubDirectory">dummy_file_for_subdirectory.dummy</Input>
```

Comment:
We have the option to delete these files, though right? If the file is read-only can we create a symbolic link to it? In case the file is large, it would allow us to bypass some overhead and save on storage. If we cannot guarantee the file is read-only, then all of this is irrelevant.